evdev/device.py

```diff
@@ -259,6 +263,37 @@ def fileno(self):
 
         return self.fd
 
+
+    if asyncio is not None:
+        def read_async(self, loop=None):
+            '''
+            Coroutine to asynchronously wait for an input event. 
+            
+            Use it with the asyncio module
+            '''
+
+            future = asyncio.Future()
+
+            # no need to wait if data is already available
+            fast_read = self.read_one()
```

Comment:
Could this be made to work with `self.read()` instead of `self.read_one()`? My guess is that the packing/unpacking overhead of calling `read_one() | _input.device_read()` is going to add up quickly.